### ISSUE TITLE
Image color filter (Brownie) output mismatch in version 1.7.22 vs 2.3.4 (latest)

### ISSUE THREAD

**Issue by shaktippatel** [fabricjs/fabric.js#5151]
Image color filter (Brownie) output mismatch in version 1.7.22 vs 2.3.4 (latest)

Matrix
`[0.59970,0.34553,-0.27082,0,0.186,-0.03770,0.86095,0.15059,0,-0.1449,0.24113,-0.07441,0.44972,0,-0.02965,0,0,0,1,0]`

## Version 1.7.22 with **ColorMatrix** Filter
[JSFIDDLE 1.7.22](http://jsfiddle.net/shaktipatel/pge8jay1/)

## Version 2.3.4 with **ColorMatrix** Filter
[JSFIDDLE 2.3.4](http://jsfiddle.net/shaktipatel/ruLydzsn/)

## Version 2.3.4 with **Brownie** Filter
[JSFIDDLE 2.3.4](http://jsfiddle.net/shaktipatel/vfdjoLgm/)
## Steps to reproduce
Upload any image in all jsfiddle and check output
## Expected Behavior
Output should be same
## Actual Behavior
Output mismatch in version 1.7.22 vs 2.3.4 (latest)

**Comment by asturur** [comment]
Please update the fiddles with a picture that show the difference.

**Comment by shaktippatel** [comment]
Updated fiddles with a picture

**Comment by asturur** [comment]
http://jsfiddle.net/ruLydzsn/6/
http://jsfiddle.net/pge8jay1/10/
http://jsfiddle.net/vfdjoLgm/8/

**Comment by shaktippatel** [comment]
Yes but , In latest fabric version 2.* I think you forgot to update matrices (Brownie, Vintage, Kodachrome, Technicolor, Polaroid, Sepia and BlackWhite)

**Comment by asturur** [comment]
i inserted them in 2.0

Do they have values bigger than 1 inside?

**Comment by shaktippatel** [comment]
No

**Version: 1**
[0.59970, 0.34553, -0.27082, 0, 0.000729411765, -0.03770, 0.86095, 0.15059, 0, -0.000568235294, 0.24113, -0.07441, 0.44972, 0, -0.00011627451, 0, 0, 0, 1, 0]

**Version: 2 (After replace 4 , 9, 14 and 19 position)**
[0.59970,0.34553,-0.27082,0,0.186,-0.03770,0.86095,0.15059,0,-0.1449,0.24113,-0.07441,0.44972,0,-0.02965,0,0,0,1,0]

then any reason not to use second array? 

or both are correct?

**Comment by asturur** [comment]
i may just have inserted them wrongly in 1.7. i will check again
